### ISSUE TITLE
Need Windows CI

### ISSUE THREAD

**Issue by erikd** [simonmar/happy#124]
It would be nice if someone who knows Windows (ie not me and not @simonmar) could provide an `appveyor.yml` file so issues like #119 can be prevented.

**Comment by hvr** [comment]
I'm on it!

**Comment by hvr** [comment]
With a bit of fiddling I was able to port the .travis.yml script to a .appveyor.yml script. It's mostly the same, except for a different setup fetching ghc+cabal from the Chocolately package repo (as opposed to fetching Linux binaries from my Ubuntu PPA)

**Comment by simonmar** [comment]
@hvr do I need to do anything other than enable AppVeyor on this repo?  @OlivierSohn are you OK with me merging this instead of yours? It looks like it tests more combinations.

**Comment by hvr** [comment]
@simonmar yeah, just enable AppVeyor; I'm actually testing an extended version that tests even more combinations, which is exposing some weird issue which I'm not sure yet if it's `happy`'s fault or GHC's

**Comment by erikd** [comment]
Does anyone still *use* 32 bit Windows?

**Comment by OlivierSohn** [comment]
@erikd in my case I was only using it by default, because the appveyor script I started with was using it :) using x64 fixed the issue!

**Comment by hvr** [comment]
@simonmar btw, a workaround that would prevent `happy` from failing to install on 32bit Win32 is to avoid the custom `Setup.hs` which directly invokes `ghc -cpp -E` for template-expansion, and instead use `build-type: Simple` (similiar to the recent change in `alex`)
